packages/functional-tests/tests/syncV3/signUp.spec.ts

```diff
@@ -0,0 +1,148 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+
+const password = 'passwordzxcv';
+const incorrectPassword = 'password123';
+let email;
+
+test.describe('Firefox Desktop Sync v3 sign up', () => {
+  test.beforeEach(async ({ pages: { login } }) => {
+    test.slow();
+    email = login.createEmail('sync{id}');
+    await login.clearCache();
+  });
+
+  test('sign up', async ({
+    target,
+    page,
+    pages: { login, signinTokenCode, connectAnotherDevice },
+  }) => {
+    await page.goto(
+      `${target.contentServerUrl}?context=fx_desktop_v3&service=sync&action=email`,
+      { waitUntil: 'networkidle' }
+    );
+    await login.setEmail(email);
+    await signinTokenCode.clickSubmitButton();
+
+    // Verify the email is correct
+    expect(await login.getPrefilledEmail()).toMatch(email);
+
+    // Passwords do not match should cause an error
+    await login.setPassword(password);
+    await login.confirmPassword(incorrectPassword);
+    await login.setAge('21');
+    await signinTokenCode.clickSubmitButton();
+
+    // Verify the error message
+    expect(await login.getTooltipError()).toMatch('Passwords do not match');
+
+    // Fix the error
+    await login.confirmPassword(password);
+    await login.submit();
+    await login.fillOutSignUpCode(email);
+    expect(await connectAnotherDevice.fxaConnected.isVisible()).toBeTruthy();
+  });
+
+  test('coppa disabled', async ({
+    target,
+    page,
+    pages: { login, connectAnotherDevice },
+  }) => {
+    const query = { coppa: 'false' };
+    const queryParam = new URLSearchParams(query);
+    await page.goto(
+      `${
+        target.contentServerUrl
+      }?context=fx_desktop_v3&service=sync&action=email&${queryParam.toString()}`,
```

Comment:
nit: not sure if it's any clearer to move these query params up to L54 above and set them all in one place, or if that makes it less obvious what's being tested/important.

```js
    const query = { coppa: 'false' };
    const queryParam = new URLSearchParams(query);
    // `${target.contentServerUrl}?context=fx_desktop_v3&service=sync&action=email&${queryParam.toString()}`
```

versus

```js
    const query = { coppa: 'false', context: 'fx_desktop_v3', service: 'sync', action: 'email' };
    const queryParam = new URLSearchParams(query);
    // `${target.contentServerUrl}?${queryParam.toString()}`
```